### ISSUE TITLE
spcomp exit code 3221225477 - no eval on "else if {"

### ISSUE THREAD

**Issue by JoinedSenses** [alliedmodders/sourcemod#970]
# Help us help you
  - [X] I have checked that my issue [doesn't exist yet](https://github.com/alliedmodders/sourcemod/issues).
  - [X] I have tried my absolute best to reduce the problem-space and have provided the absolute smallest test-case possible.
  - [X] I can always reproduce the issue with the provided description below.

# Environment
  * Operating System version:  Win 10
  * Game/AppID (with version if applicable): NA
  * Current SourceMod version: 1.10
  * Current SourceMod snapshot: 6380
  * Current Metamod: Source snapshot: NA
  - [X] I have updated SourceMod to the [latest version](https://www.sourcemod.net/downloads.php) and it still happens.
  - [ ] I have updated SourceMod to the [latest snapshot](https://www.sourcemod.net/downloads.php?branch=dev) and it still happens.
  - [NA] I have updated SourceMM to the [latest snapshot](https://sourcemm.net/downloads.php?branch=dev) and it still happens.

# Description
spcomp exit code 3221225477 with no warning for missing parenthesis of `else if {`

# Problematic Code (or Steps to Reproduce)
  ```
#pragma semicolon 1
#pragma newdecls required

#include <sourcemod>

ConVar g_cvarTest;
bool g_bTest;

public void OnPluginStart() {
	HookEvent("item_pickup", PickUp, EventHookMode_Pre);
}

public Action event_ItemPickup(Event event, const char[] name, bool dontBroadcast) {
	if (g_cvarTest == null) {
		return Plugin_Handled;
	}
	else if {
		int client = GetClientOfUserId(event.GetInt("userid"));
		if (client < 0) {
			return Plugin_Handled;
		}
	}
 
	return Plugin_Continue;
}
  ```

**Comment by Batfoxkid** [comment]
Trying to use `else`?

**Comment by JoinedSenses** [comment]
@Batfoxkid No, just a simple test case of an `else if()` not catching missing parenthesis, and instead exiting

**Comment by Batfoxkid** [comment]
Ahh.

**Comment by Headline** [comment]
also occurs with empty `while`'s or `do {} while`'s.

Once a second one is added, it seems to error as expected.

**Comment by CrazyHackGUT** [comment]
I think, it should be moved to [another repository](https://github.com/alliedmodders/sourcepawn).
